src/classes/PS_Request.cls

```diff
@@ -69,6 +72,8 @@ public inherited sharing class PS_Request {
     public static String ENDPOINT_PURCHASE = '/v1/payments/verified/purchase';
     public static String ENDPOINT_COMMITMENT = '/v1/payments/verified/commitments';
     public static String ENDPOINT_COMMITMENT_CANCEL = '/v1/payments/verified/commitments/{0}/cancel';
+    public static String ENDPOINT_CREATE_CAPTURE_GROUP = '/v1/payments/verified/batch';
+    public static String ENDPOINT_ADD_TO_CAPTURE_GROUP = 'v1/payments/verified/batch/{0}/add';
```

Comment:
@dfuller114 - Just a minor thing, but the endpoint here is missing a forward slash before "v1"

Reply:
fixed!